### PR TITLE
Updated cfp, coc and location pages, and added favicon

### PR DESCRIPTION
**Pull request by 571530**
Also progress on talks&speaker and schedule

### CODE REVIEW

File: styling/scheduleStyles.scss [hunk 1]
```diff
@@ -45,40 +59,245 @@ div.schedule {
...
+        .room {
+          @extend .hide-schedule;
+        }
+      }
+    }
+  }
+}
+
+.room-container {
+  flex-shrink: 1;
```
Comment (Maritt) [inline styling/scheduleStyles.scss]: should probably use the following notation for shrink/grow/basis: `flex: none | [ <'flex-grow'> <'flex-shrink'>? || <'flex-basis'> ]`

source: https://css-tricks.com/snippets/css/a-guide-to-flexbox/

File: styling/styling.scss [hunk 2]
```diff
@@ -36,39 +33,84 @@ h1 {
...
+    display: none !important;
+  }
+}
+.hide-large {
+  @media screen and (min-width: $break-large) {
+    display: none !important;
+  }
+}
+.show-small {
+  display: none !important;
```
Comment (Maritt) [inline styling/styling.scss]: using !important notation is often not the most wanted situation. And often indicates that styling has become hard due to overwriting of styles.

File: styling/variables.scss [hunk 3]
```diff
@@ -1,13 +1,38 @@
...
-$primary-color: #28334a;
-$button-color: #e4e2db;
-$secondary-color: #75787b;
+$break-schedule: 1086px;
+
+$header-height: 100px;
+$min-header-height: 44px;
+$header-bottom-margin: 50px;
+
+$primary-color: #1d2e46;
```
Comment (Maritt) [inline styling/variables.scss]: this color is declared in several places, could the color be reused instead of recreated?

File: tsconfig.json [hunk 4]
```diff
@@ -24,6 +24,6 @@
   "include": [
     "next-env.d.ts",
     "**/*.ts",
-    "**/*.tsx"
+    "**/*.tsx", "pages/_document.tsx"
```
Comment (Maritt) [inline tsconfig.json]: new line?

File: pages/components/Layout.tsx [hunk 5]
```diff
@@ -5,16 +5,25 @@ import Filter from "./Filter";
...
-class Layout extends React.Component<any, any> {
+class Layout extends React.Component<IProps, any> {
   render() {
     return (
       <div className="layout">
         <div className="menuAndHeader">
           <div className="header">
-            {!this.props.hideLogo && (<Link href="/">
-              <img id="KDSlogo" src="../static/KDSsymbol.svg" />
+            {this.props.hideLogo !== true && (<Link href="/">
```
[the remark this answers is not in the document]
Comment (571530) [reply pages/components/Layout.tsx]:
```typescript
hideLogo?: 'small' | 'large' | boolean
 ```
hideLogo can be the breakpoint for hiding the logo, !hideLogo would give false for 'small' and 'large'

**Pull request merged**